timetagger/__main__.py

```diff
@@ -117,6 +117,7 @@ async def api_handler(request, path):
     # Authenticate and get user db
     try:
         auth_info, db = await authenticate(request)
+        await validate_auth(request, auth_info)
```

Comment:
Could you please wrap this in a `if not config.proxy_auth_enabled:` (and remove that if from the function)?

Reply:
Done